app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/header", title: "LOGIN", prev: root_path, value: "Log in" %>
+
+<div class="bg-[#f5f6f8] h-full pt-6">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email, class: "sr-only" %>
+    <%= f.email_field :email, autofocus: true, placeholder: "Email", class: "w-full border border-[#E9EDF4] py-3 px-5 text-base placeholder-[#ACB6BE] outline-none focus-visible:shadow-none focus:border-primary" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password, class: "sr-only" %>
+    <%= f.password_field :password, placeholder: "Password", class: "w-full border border-[#E9EDF4] py-3 px-5 text-base placeholder-[#ACB6BE] outline-none focus-visible:shadow-none focus:border-primary" %>
+  </div>
+
+<% end %>
+
+<div class="flex flex-col items-center w-full gap-4 my-2 text-base">
+<span class="hover:underline hover:text-primary"><%= link_to "Forgot your password?", new_password_path(resource_name) %></span>
```

Comment:
current as the previous reviewer mentioned the deployed page is not working and can be viewed on this page 

![image](https://user-images.githubusercontent.com/60612329/195927449-2ff34fc0-9f6f-45d8-9e9c-a7fdd9e6966f.png)

Reply:
I have updated the link to [LIve Link](https://peaceful-river-33336.herokuapp.com/)